src/field.rs

```diff
@@ -922,15 +941,6 @@ mod tests {
             assert_eq!(product, expected_product);
             assert_eq!(hash_helper(product), expected_hash);
         }
-
-        // Construct an element from a number that needs to be reduced, and test comparisons on it,
-        // confirming that FieldParameters::montgomery() reduced it correctly.
-        let p = F::from(int_modulus);
-        assert_eq!(p, zero);
-        assert_eq!(hash_helper(p), hash_helper(zero));
-        let p_plus_one = F::from(int_modulus + F::Integer::try_from(1).unwrap());
-        assert_eq!(p_plus_one, one);
-        assert_eq!(hash_helper(p_plus_one), hash_helper(one));
```

Comment:
Is this test not valuable anymore?

Reply:
No because it's now an error to try to convert an unreduced integer to a field element.